packages/plugins/src/plugins/eval.ts

```diff
@@ -48,6 +48,7 @@ export const evalPlugin: PluginFunction = (bot) => {
             msg.info.rawContent.replace(/<br>/g, '\n').replace(/<.+>/g, '')
         );
         if (
+            // @ts-ignore FIXME: can RegisterHandler accept async callbacks?
```

Comment:
Yes, it should be able to

Reply:
Thanks for reassuring! Resolved in 975cba6cb2807af0c4c5aab4d96cc388fc51db50